### ISSUE TITLE
[BUG] Motherboard audio profile changes to duplex when recording

### ISSUE THREAD

**Issue by GenocideStomper** [dec05eba/gpu-screen-recorder-issues#58]
Note: This is a low priority "bug" as it doesn't break anything.

**Describe the bug**
But I have a USB mic which is its own audio device in pavucontrol. In pavucontrol I set the motherboard audio device to the "Output" profile, since I have no need for the motherboard's microphone input device.
When I start gpu-screen-recorder, my motherboard device profile gets switched to "Duplex" (i.e. I get an extra input device for the motherboard's microphone). When I stop gpu-screen-recorder, the profile gets reverted to the original.
KDE Plasma's OSD shows the switch in profile (see screenshots).

**To Reproduce**
`gpu-screen-recorder -w screen-direct-force -k h265 -r 180 -c mp4 -f 60 -a default_output -o .`

**Screenshots**
When starting gpu-screen-recorder and it switches to duplex:
![image](https://github.com/user-attachments/assets/86766495-1ab4-43ec-9a04-a0fbb925c34e)

Input devices during recording:
![image](https://github.com/user-attachments/assets/f9fa34a1-710b-4c95-9874-ae86f4ba202d)

When stopping gpu-screen-recorder and it switches to output:
![image](https://github.com/user-attachments/assets/94ac958d-8ad8-443d-927d-a20fa6e7e639)

Input devices normally:
![image](https://github.com/user-attachments/assets/1bc29768-4bcf-450b-bb66-80ff4a8f1966)

**Desktop (please complete the following information):**
 - X11 or Wayland: X11
- Desktop environment/Window Manager: KDE Plasma 6
 - Distro: EndeavourOS
 - GPU: Nvidia GTX 1080
 - Version (latest flatpak version, aur or source): v4.2.3, AUR r856.7428f9f-1

**Additional context**
Switch doesn't happen if `-a` argument is omitted.
Switch also happens if instead of `-a default_output`, `-a "$(pactl get-default-sink).monitor"` is used.
Pipewire is used.

**Comment by dec05eba** [comment]
Im pretty sure its unrelated to gpu screen recorder itself. Did you try if the same thing happens for example if you use simple screen recorder instead and record the same audio device?

**Comment by GenocideStomper** [comment]
simplescreenrecorder does not have the issue, however OBS installed through flatpak does. Very strange, but I'll close this since it's not gpu-screen-recorder specific. Thank you!

**Comment by dec05eba** [comment]
I also have some weird issue related to audio recording, but for me it changes profile. It might be some weird issue in pipewire. Im on arch so it might be related.

**Comment by dec05eba** [comment]
This might be the issue @GenocideStomper https://gitlab.freedesktop.org/pipewire/wireplumber/-/issues/715

**Comment by dec05eba** [comment]
I downgraded wireplumber to 0.55 which fixed it for me

**Comment by GenocideStomper** [comment]
> This might be the issue @GenocideStomper https://gitlab.freedesktop.org/pipewire/wireplumber/-/issues/715 there is also a linked issue with profile automatically switching to audio stereo duplex

You legend, thank you very much!

I did what basically this comment said https://gitlab.freedesktop.org/pipewire/wireplumber/-/issues/634#note_2380391 and created the path + file and restarted. Seems to have fixed the issue!

Sorry for bringing up the issue now. I finally upgraded my installation after 8+ months because Plasma 6 was not stable enough for me, and I didn't want to partially upgrade.
